examples/api/lib/cupertino/date_picker/cupertino_date_picker.0.dart

```diff
@@ -0,0 +1,169 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for CupertinoDatePicker
```

Comment:
This line is the only comment in the example, and it doesn't really say anything. Since this is a sample meant to be read by people learning Flutter, it would be good to annotate the relevant portions of the code with comments. Remember that this might be the first piece of Flutter code that they might ever have seen.  It doesn't need to be littered with comments, but it should at least talk about the concepts relevant to the `CupertinoDatePicker`.  Talk about why you used which arguments in the date picker, why you have a separate private class for the items, etc. Some of that should be in the description at the point where it is referred to in the API docs, but pointing out important parts of the example with comments is also useful. Otherwise, the new users will not know which parts of the example are relevant to this widget, and which are just boilerplate.

Reply:
Thanks for this, I understand the importance of this much better now. I'm no writer but I tried to provide my explanation here. Let me know if you think this is the right direction.